crates/swc/tests/tsc-references/awaitCallExpression4_es2017_es2015.1.normal.js

```diff
@@ -7,7 +7,7 @@ function func() {
 function _func() {
     _func = _async_to_generator(function*() {
         before();
-        var b = yield pfn(a, a, a);
+        var b = (yield pfn)(a, a, a);
```

Comment:
@magic-akari I don't know what these test files are but just double checking - the semantic change here is expected, right? Before it yielded the function result, now it yields `pfn` and then calls it

Reply:
Yes. It's a bugfix.